mmv1/third_party/terraform/tests/data_source_iap_client_test.go

```diff
@@ -0,0 +1,73 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIapClient_Datasource_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        getTestOrgFromEnv(t),
+		"org_domain":    getTestOrgDomainFromEnv(t),
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccIapClientDatasourceConfig(context),
+				Check: resource.ComposeTestCheckFunc(
+					checkDataSourceStateMatchesResourceStateWithIgnores(
+						"data.google_iap_client.project_client",
+						"google_iap_client.project_client",
+						map[string]struct{}{
+							"brand": {},
+						},
+					),
+				),
+			},
+		},
+	})
+}
+
+func testAccIapClientDatasourceConfig(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_project" "project" {
+  project_id = "tf-test%{random_suffix}"
+  name       = "tf-test%{random_suffix}"
+  org_id     = "%{org_id}"
+}
+
+resource "google_project" "project" {
```

Comment:
It looks like this needs a different name. I'm seeing the failure:

```
TestAccIapClient_Datasource_basic: provider_test.go:270: Step 1/1 error: Error running pre-apply refresh: 2021/02/10 23:33:21 [DEBUG] Using modified User-Agent: Terraform/0.12.30 HashiCorp-terraform-exec/0.12.0
              
              Error: Duplicate resource "google_project" configuration
              
                on config792785144/terraform_plugin_test.tf line 8:
                 8: resource "google_project" "project" {
              
              A google_project resource named "project" was already declared at
              config792785144/terraform_plugin_test.tf:2,1-36. Resource names must be unique
              per type in each module.
```

Reply:
Fixed, this is a rough one to test cleanly.